src/common/commonutils/UserUtils.c

```diff
@@ -819,49 +816,37 @@ int CheckNoDuplicateUidsExist(char** reason, OsConfigLogHandle log)
     return status;
 }
 
-int RemoveUser(SimplifiedUser* user, bool removeHome, OsConfigLogHandle log)
+static int RemoveUser(const char* username, const uid_t uid, const gid_t gid, const char* home, OsConfigLogHandle log)
```

Comment:
>const char* username, const uid_t uid, [](http://example.com/codeflow?start=22&length=38)

Same feedback as before: why not keep the SimpliedUser argument, basically keep the RemoveUser declaration as used before just without the removeHome argument?

This would minimize the changes in this PR even further. Plus, most functions like this use the SimplifiedUser argument instead of separate user name and uid.